SessionUIKit/Style Guide/Constants+Apple.swift

```diff
@@ -11,4 +11,8 @@ public extension Constants {
     
     // MARK: - Names
     static let store_name = "App Store"
+    static let platform_name = "iOS"
+    
+    // MARK: -
+    static let review_prompt_availability_version = "2.14.1"
```

Comment:
Can you please move this to be a static variable in `AppReviewPromptModel`? (It's unlikely that it would ever be used elsewhere so it can live in there)

Additionally we probably need to add a `TODO` or a task to ensure we update this value prior to the release (as `2.14.1` is already released, and `2.14.2` is probably going to to out early next week which might be too soon for this PR to get merged - will probably be `2.14.3` pending QA)